docs/output.md

```diff
@@ -0,0 +1,339 @@
+# NGI-RNAseq Output
+
+NGI-RNAseq is the new RNA-seq Best Practice pipeline used by the [National Genomics Infrastructure](https://ngisweden.scilifelab.se/) at [SciLifeLab](https://www.scilifelab.se/platforms/ngi/) in Stockholm, Sweden.
+
+This document describes the output produced by the pipeline.
+
+## Pipeline overview:
+The pipeline is built using [Nextflow](https://www.nextflow.io/)
+and processes data using the following steps:
+
+* [FastQC](#fastqc) - read quality control
+* [TrimGalore](#trimgalore) - adapter trimming
+* [STAR](#star) - alignment
+* [RSeQC](#rseqc) - RNA quality control metrics
+   - [BAM stat](#bam-stat)
+   - [Infer experiment](#infer-experiment)
+   - [Junction saturation](#junction-saturation)
+   - [RPKM saturation](#rpkm-saturation)
+   - [Read duplication](#read-duplication)
+   - [Inner distance](#inner-distance)
+   - [Gene body coverage](#gene-body-coverage)
+   - [Read distribution](#read-distribution)
+   - [Junction annotation](#junction-annotation)
+* [dupRadar](#dupradar) - technical / biological read duplication
+* [Preseq](#preseq) - library complexity
+* [featureCounts](#featurecounts) - gene counts, biotype counts, rRNA estimation.
+* [StringTie](#stringtie) - FPKMs for genes and transcripts
+* [Sample_correlation](#Sample_correlation) - create MDS plot and sample pairwise distance heatmap / dendrogram
+* [MultiQC](#multiqc) - aggregate report, describing results of the whole pipeline
+
+## FastQC
+[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your reads. It provides information about the quality score distribution across your reads, the per base sequence content (%T/A/G/C). You get information about adapter contamination and other overrepresented sequences.
+
+For further reading and documentation see the [FastQC help](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
+
+> **NB:** The FastQC plots displayed in the MultiQC report shows _untrimmed_ reads. They may contain adapter sequence and potentially regions with low quality. To see how your reads look after trimming, look at the FastQC reports in the `trim_galore` directory.
+
+**Output directory: `results/fastqc`**
+
+* `sample_fastqc.html`
+  * FastQC report, containing quality metrics for your untrimmed raw fastq files
+* `sample_fastqc.zip`
+  * zip file containing the FastQC report, tab-delimited data file and plot images
```

Comment:
Are these supposed to be lists? They don't show up like that for me. Maybe it works on github 😕 . Same thing for virtually all the other programs paragraphs.